### PR TITLE
[API] GET PrayerRequests queries

### PR DESCRIPTION
**Pull request by IsaacHardy**
## DESCRIPTION
Closes SYS-4633 and SYS-4629
### What does this PR do, or why is it needed?
Adds the ability to GET public prayer requests. Also by campus and id.

### What design trade-offs/decisions were made?
n.a.

### How do I test this PR?
1. Boot up the API
2. Login as your user and store your auth token
3. Query for getPublicPrayerRequests
```
query {
  getPublicPrayerRequests {
    id
    firstName
    lastName
    text
    createdByPersonAliasId
    campusId
    categoryId
    flagCount
    prayerCount
  }
}
```
4. Query for getPublicPrayerRequestsByCampus
```
query {
  getPublicPrayerRequestsByCampus (campusId: 16) {
    id
    firstName
    lastName
    text
    createdByPersonAliasId
    campusId
    categoryId
    flagCount
    prayerCount
  }
}
```
5. Query for getPrayerRequestById
```
query {
        node (id: "PrayerRequest:b36e55d803443431e96bb4b5068147ec") {
          __typename
          ... on PrayerRequest {
            id
            firstName
            lastName
            text
            createdByPersonAliasId
            campusId
            categoryId
            flagCount
            prayerCount
          }
        }
      }
```
6. Query for getCurrentPersonPrayerRequests
```
query {
  getCurrentPersonPrayerRequests {
    id
    firstName
    lastName
    text
    createdByPersonAliasId
    campusId
    categoryId
    flagCount
    prayerCount
  }
}
```
## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [tag-issues-here]
- [x] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/src/tabs/connect/Connect.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ const Title = styled(({ theme }) => ({
...
-  size: theme.sizing.baseUnit * 2.25,
-  marginBottom: theme.sizing.baseUnit,
-  fill: theme.colors.primary,
-}))(Icon);
+// const BrandIcon = withTheme(({ theme }) => ({
+//   name: 'brand-icon',
+//   size: theme.sizing.baseUnit * 2.25,
+//   marginBottom: theme.sizing.baseUnit,
+//   fill: theme.colors.primary,
+// }))(Icon);
```
Comment (redreceipt) [inline packages/apolloschurchapp/src/tabs/connect/Connect.js]: what does this file have to do with this PR?

**Comment by redreceipt** [comment]
@IsaacHardy I think it would be better to use the guid returned to query by ID, not the unique table ID.

File: packages/apolloschurchapp/src/tabs/connect/Connect.js (hunk 1 above)
Comment (IsaacHardy) [reply packages/apolloschurchapp/src/tabs/connect/Connect.js]: It has to do with our App. It causes lint errors that weren't resolved in the master branch.

**Comment by redreceipt** [comment]
@IsaacHardy `getPrayerRequestById` is still showing up in my schema on GraphQL Playground. Is there some cache I can clear somewhere?

screen shot 2019-03-05 at 2 57 09 pm

**Comment by IsaacHardy** [comment]
@redreceipt Refresh the browser.

**Comment by redreceipt** [comment]
that did it :)
